source/develop/release-management/features/sla/vm-affinity.html.md

```diff
@@ -89,15 +89,15 @@ affinity_group_members: affinity_group_id (foreign key to affinity_groups + dele
 
 ### UI
 
-![Relations sub tab under clusters](/images/wiki/relation-cluster.png "Relations sub tab under clusters")
+![Relations sub tab under clusters](/images/wiki/Relation-cluster.png "Relations sub tab under clusters")
 
 An overview of all Affinity Groups in the cluster.
 
-![Relations sub tab under clusters](/images/wiki/relation-vm.png "Relations sub tab under clusters")
+![Relations sub tab under clusters](/images/wiki/Relation-vm.png "Relations sub tab under clusters")
 
 All Affinity Groups associated with a single virtual machine.
 
-![Add/Edit Relation Dialog](/images/wiki/relation-dialog.png "Add/Edit Relation Dialog")
+![Add/Edit Relation Dialog](/images/wiki/Relation-dialog.png "Add/Edit Relation Dialog")
 
 Dialog for adding or editing a single relation, specifying Name, polarity, enforcement mode, and members (virtual machines). The virtual machine drop-down menu should support typing (a suggestion combo box) for quick selection.
```

Comment:
this was a description for the picture above (if the description still needs editing then it will change)